password_security/models/res_users.py

```diff
@@ -60,6 +61,9 @@ def password_match_message(self):
         if company_id.password_special:
             message.append('\n* ' + 'Special character (At least ' + str(
                 company_id.password_special) + ' character)')
+        if company_id.password_not_similar_name_login:
+            message.append(
+                '\n* ' + 'Must not be similar to the user\'s login or name.')
```

Comment:
Yikes! Why not `"\n* Must not be similar to the user's login or name."`?

Translate it please.